purchases/src/androidTestIntegrationTest/kotlin/com/revenuecat/purchases/BasePurchasesIntegrationTest.kt

```diff
@@ -107,18 +111,38 @@ open class BasePurchasesIntegrationTest {
         activityScenarioRule.scenario.onActivity(block)
     }
 
-    protected fun simulateSdkRestart(context: Context, forceServerErrors: Boolean = false) {
+    protected fun simulateSdkRestart(
+        context: Context,
+        entitlementVerificationMode: EntitlementVerificationMode? = null,
```

Comment:
`EntitlementVerificationMode` already has `DISABLED`, so does this need to be `null`?
Oh I guess it's to be able to determine if it's been explicitly set? It might be useful to have a comment explaining that.
Update: oh just realized this is in tests, so makes sense.

Reply:
Yeah, it's pretty much a way to make sure it's explicitly set 👍